### PR TITLE
feat: add nest js rest api starter

### PR DESCRIPTION
**Pull request by ada-krupa**

### CODE REVIEW

File: template/nestjs-rest-api/base/yarn.lock
Comment (Brentlok) [inline template/nestjs-rest-api/base/yarn.lock:1]: We should allow user to use any package manager, so we shouldn't include yarn.lock

**Review by Brentlok — CHANGES_REQUESTED** [review]

File: template/nestjs-rest-api/base/package.json [hunk 1]
```diff
@@ -0,0 +1,98 @@
+{
+  "name": "api-starter",
+  "version": "1.0.0",
+  "author": "Codegate",
```
Comment (adrianboratyn) [inline template/nestjs-rest-api/base/package.json]: shouldn't be Codemask? 🤔

File: template/nestjs-rest-api/base/yarn.lock
Comment (WadasGrzegorz) [inline template/nestjs-rest-api/base/yarn.lock:1]: @Brentlok are you sure? what about scripts that we are using for build etc?

**Comment by Brentlok** [comment]
@WadasGrzegorz about script in package.json - we could rename "yarn" to whatever package manager someone is  using while copying the template. But when it comes to yarn.lock, you're right. I guess we will force usage of yarn then

**Review by Brentlok — APPROVED** [review]

**Pull request merged**